modeltrans/fields.py

```diff
@@ -83,10 +87,22 @@ def contribute_to_class(self, cls, name):
     def db_type(self, connection):
         return None
 
+    def get_instance_fallback_chain(self, instance, language):
+        default = get_fallback_chain(language)
+
+        i18n_field = instance._meta.get_field("i18n")
+        if i18n_field.fallback_language_field:
+            record_fallback_language = get_instance_field_value(
```

Comment:
@jieter ik snap nog niet helemaal waarom, maar dit gaat fout in de challenge create view voor het ChallengeContentForm: hij probeert denk ik voor translation form fields (text_nl etc) de fallback language op te halen, en probeert dat dan te doen via challenge__language_code (omdat dat dat de fallback_language_field is). MAAR hij vind de challenge niet: app.challenges.models.ChallengeContent.challenge.RelatedObjectDoesNotExist: ChallengeContent has no challenge. 

Ik had even in de challenge create view getest, en de voor de challengecontent instance die daar aangemaakt wordt om naar het form mee te sturen, daarvan krijg je met getattr dezelfde foutmelding. Dus van de "nog niet opgeslagen" instance werkt getattr dus niet, en gaat het hier dus mis met de modelform.

Reply:
Ik heb nog wat verbeteringetjes doorgevoerd, waardoor dit was robuuster zou moeten werken, wil je het nog een keer proberen?

Bij de volgende crash graag de hele stack-trace delen, dan kan ik beter zien wat er precies mis gaat.

Reply:
Dit lijkt opgelost